docs/src/main/paradox/java/http/server-side/server-https-support.md

```diff
@@ -0,0 +1 @@
+../../../scala/http/server-side/server-https-support.md
```

Comment:
While I like this change (and agree dropping `server-` makes sense), this will break external links to these pages in the documentation. One way to fix that might be to maintain a `.htaccess` file with redirects in our documentation codebase (https://github.com/lightbend/paradox/issues/131)